packages/appkit/src/client.ts

```diff
@@ -470,11 +474,34 @@ export class AppKit {
       adapter => adapter.chainNamespace === ConstantsUtil.CHAIN.EVM
     )
 
+    // Only set the analytics state if it's not already set through the SDK config
+    if (options.features?.analytics === undefined) {
+      const projectCloudConfig = await ApiController.fetchProjectConfig()
+      OptionsController.setFeatures({ analytics: projectCloudConfig?.isAppKitAuthEnabled })
+
+      if (options.enableAuth === undefined) {
```

Comment:
can we use default params instead of checking against `undefined` ?

Reply:
So we're just removing the dependency on getting the config from Cloud ?
The initial product requirement was, if `enableAnalytics` is not set on the AppKit config, then check the Cloud config. By introducing a default value, you're breaking that flow.

Reply:
I see, a bit of a weird flow imo but if that was the product decision that's ok